src/Promise.php

```diff
@@ -175,16 +202,21 @@ private function unwrap($promise): PromiseInterface
         return $promise;
     }
 
-    private function call(callable $callback): void
+    private function call(callable $cb): void
     {
+        // Explicitly overwrite argument with null value. This ensure that this
+        // argument does not show up in the stack trace in PHP 7+ only.
+        $callback = $cb;
+        $cb = null;
+
         // Use reflection to inspect number of arguments expected by this callback.
         // We did some careful benchmarking here: Using reflection to avoid unneeded
         // function arguments is actually faster than blindly passing them.
         // Also, this helps avoiding unnecessary function arguments in the call stack
         // if the callback creates an Exception (creating garbage cycles).
-        if (\is_array($callback)) {
+        if (is_array($callback)) {
```

Comment:
Should probably not be changed, see also other occurrences :+1:

Reply:
Will fix that, but there also will be a follow up changing this like this to `use function is_array;`.

Reply:
@WyriHaximus Not sure I like this suggestion, but let's continue this discussion in a dedicated ticket :+1:

Reply:
@clue Let me work it out and give a good reasoning behind it first ;)

Reply:
@clue fixed this as well now